packages/components/pie-modal/src/index.ts

```diff
@@ -224,6 +232,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
                 @click="${() => { this._backButtonClicked = true; this.isOpen = false; }}"
                 variant="ghost-secondary"
                 class="c-modal-backBtn"
+                aria-label="${this.ariaLabels?.backButton || 'Back'}"
```

Comment:
same here, not having an aria-label at all is still better than having an incorrect/confusing label. what if our component library is only used for Hebrew and Arabic sites?  this will create unnecessary noise and confusion to end users.

Reply:
Fixed.